pandas/_libs/tslibs/period.pyx

```diff
@@ -1408,6 +1408,23 @@ cdef class _Period(object):
 
     @property
     def qyear(self):
+        """
+        Get the year component of the Period.
+        
+        Returns
+        -------
+        int
+
+        See Also
+        --------
+        Period.qyear :Return the year of the date
```

Comment:
add pqyear in See Also section

Reply:
Can you fix the spacing here? Space after the `:`